### ISSUE TITLE
 High quality checkpoint in Spanish?

### ISSUE THREAD

**Issue by acclayer7** [rhasspy/piper#628]
I am using the ald medium model for ES_MX

https://huggingface.co/datasets/rhasspy/piper-checkpoints/tree/main/es/es_MX/ald

I would like to continue training with a checkpoint. Is there any high-quality repository for ES_MX Spanish?
Thanks!

**Comment by synesthesiam** [comment]
No, sorry. I never trained a high quality version of that model :/
